Puppetfile.lock

```diff
@@ -0,0 +1,122 @@
+    puppet-systemd (3.5.1)
+      puppetlabs-inifile (>= 1.6.0, < 6.0.0)
+      puppetlabs-stdlib (>= 4.13.1, < 9.0.0)
+    puppet-trusted_ca (4.0.0)
+      puppetlabs-stdlib (>= 4.13.0, < 8.0.0)
+    puppetlabs-apache (6.5.1)
+      puppetlabs-concat (>= 2.2.1, < 8.0.0)
+      puppetlabs-stdlib (>= 4.13.1, < 9.0.0)
+    puppetlabs-apt (8.3.0)
+      puppetlabs-stdlib (>= 4.16.0, < 9.0.0)
+    puppetlabs-concat (7.1.1)
+      puppetlabs-stdlib (>= 4.13.1, < 9.0.0)
+    puppetlabs-inifile (5.2.0)
+      puppetlabs-stdlib (>= 4.13.0, < 9.0.0)
+    puppetlabs-postgresql (7.5.0)
+      puppetlabs-apt (>= 2.0.0, < 9.0.0)
+      puppetlabs-concat (>= 4.1.0, < 8.0.0)
+      puppetlabs-stdlib (>= 4.13.1, < 9.0.0)
+    puppetlabs-stdlib (7.1.0)
+    puppetlabs-xinetd (3.3.0)
+    richardc-datacat (0.6.2)
+    theforeman-dhcp (8.0.0)
+      puppet-systemd (>= 3.1.0, < 4.0.0)
+      puppetlabs-concat (>= 1.0.0, < 8.0.0)
+      puppetlabs-stdlib (>= 4.18.0, < 8.0.0)
+    theforeman-dns (9.1.0)
+      puppetlabs-concat (>= 4.1.0, < 8.0.0)
+      puppetlabs-stdlib (>= 4.13.1, < 8.0.0)
+    theforeman-foreman (19.0.0)
+      puppet-extlib (>= 3.0.0, < 6.0.0)
+      puppet-redis (>= 5.0.0, < 9.0.0)
+      puppet-systemd (>= 2.2.0, < 4.0.0)
+      puppetlabs-apache (>= 5.5.0, < 7.0.0)
+      puppetlabs-apt (>= 2.0.0, < 9.0.0)
+      puppetlabs-concat (>= 1.0.0, < 8.0.0)
+      puppetlabs-postgresql (>= 6.5.0, < 8.0.0)
+      puppetlabs-stdlib (>= 4.25.0, < 8.0.0)
+    theforeman-foreman_proxy (20.1.0)
+      puppet-extlib (>= 3.0.0, < 6.0.0)
+      puppetlabs-stdlib (>= 4.19.0, < 9.0.0)
+      richardc-datacat (>= 0.6.0, < 1.0.0)
+      theforeman-dhcp (>= 6.1.0, < 9.0.0)
+      theforeman-dns (>= 8.1.0, < 10.0.0)
+      theforeman-foreman (>= 10.0.0, < 20.0.0)
+      theforeman-tftp (>= 3.0.0, < 8.0.0)
+    theforeman-git (7.0.0)
+      puppetlabs-stdlib (>= 4.13.0, < 8.0.0)
+    theforeman-pulpcore (5.1.0)
+      puppet-extlib (>= 3.0.0, < 6.0.0)
+      puppet-redis (>= 5.0.0, < 9.0.0)
+      puppet-systemd (>= 2.2.0, < 4.0.0)
+      puppetlabs-apache (>= 5.4.0, < 7.0.0)
+      puppetlabs-concat (>= 4.0.0, < 8.0.0)
+      puppetlabs-postgresql (>= 6.5.0, < 8.0.0)
+      puppetlabs-stdlib (>= 4.25.0, < 8.0.0)
+    theforeman-puppet (15.2.0)
+      puppet-extlib (>= 3.0.0, < 6.0.0)
+      puppet-systemd (>= 2.9.0, < 4.0.0)
+      puppetlabs-concat (>= 4.1.0, < 8.0.0)
+      puppetlabs-stdlib (>= 4.18.0, < 8.0.0)
+    theforeman-puppetserver_foreman (2.0.0)
+      puppetlabs-stdlib (>= 4.18.0, < 8.0.0)
+    theforeman-tftp (7.0.0)
+      puppetlabs-stdlib (>= 4.13.1, < 8.0.0)
+      puppetlabs-xinetd (>= 1.1.0, < 4.0.0)
+
+DEPENDENCIES
+  katello-candlepin (>= 12.1.0, < 12.2.0)
+  katello-certs (>= 14.0.0, < 14.1.0)
+  katello-foreman_proxy_content (>= 20.1.0, < 20.2.0)
+  katello-katello (>= 21.2.0, < 21.3.0)
+  katello-qpid (>= 9.1.0, < 9.2.0)
+  puppetlabs-postgresql (>= 7.0.0)
+  puppetlabs-xinetd (>= 3.3.0, < 3.4.0)
```

Comment:
It seems we will need to bump `puppetlabs-xinetd` which means bump also required for `theforeman-tftp`
~~~
Puppet 7.12.1 does not meet requirements for puppetlabs-xinetd (>= 4.7.1 < 7.0.0)
~~~

Reply:
See https://github.com/theforeman/foreman-installer/pull/733/files#r747798250